cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -0,0 +1,279 @@
+configOverride:
+#configOverride: |
+#  [global]
+#  mon_allow_pool_delete = true
+#
+#  osd_pool_default_size = 3
+#  osd_pool_default_min_size = 2
+
+# Installs a debugging toolbox deployment
+toolbox:
+  enabled: false
+  tolerations: []
+  affinity: {}
+
+monitoring:
+  # requires Prometheus to be pre-installed
+  # enabling will also create RBAC rules to allow Operator to create ServiceMonitors
+  enabled: false
+
+# All values below are taken from the CephCluster CRD
+# More information can be found at [Ceph Cluster CRD](/Documentation/ceph-cluster-crd.md)
+cephClusterSpec:
+  dataDirHostPath: /var/lib/rook
+
+  cephVersion:
+    image: ceph/ceph:v16.2.1
+    allowUnsupported: false
+  skipUpgradeChecks: false
+
+  # WaitTimeoutForHealthyOSDInMinutes defines the time (in minutes) the operator would wait before an OSD can be stopped for upgrade or restart.
+  # If the timeout exceeds and OSD is not ok to stop, then the operator would skip upgrade for the current OSD and proceed with the next one
+  # if `continueUpgradeAfterChecksEvenIfNotHealthy` is `false`. If `continueUpgradeAfterChecksEvenIfNotHealthy` is `true`, then opertor would
+  # continue with the upgrade of an OSD even if its not ok to stop after the timeout. This timeout won't be applied if `skipUpgradeChecks` is `true`.
+  # The default wait timeout is 10 minutes.
+  waitTimeoutForHealthyOSDInMinutes: 10
+
+  # The option to automatically remove OSDs that are out and are safe to destroy.
+  removeOSDsIfOutAndSafeToRemove: false
+
+  storage:
+    useAllNodes: false
+    useAllDevices: false
+    deviceFilter:
+    config:
+    # crushRoot: "custom-root" # specify a non-default root label for the CRUSH map
+    # metadataDevice: "md0" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
+    # databaseSizeMB: "1024" # uncomment if the disks are smaller than 100 GB
+    # journalSizeMB: "1024"  # uncomment if the disks are 20 GB or smaller
+    # osdsPerDevice: "1" # this value can be overridden at the node or device level
+    # encryptedDevice: "true" # the default value for this option is "false"
+    # Individual nodes and their config can be specified as well, but 'useAllNodes' above must be set to false. Then, only the named
+    # nodes below will be used as storage resources.  Each node's 'name' field should match their 'kubernetes.io/hostname' label.
+    nodes:
+  #    - name: "172.17.4.201"
+  #      devices: # specific devices to use for storage can be specified for each node
+  #      - name: "sdb"
+  #      - name: "nvme01" # multiple osds can be created on high performance devices
+  #        config:
+  #          osdsPerDevice: "5"
+  #      - name: "/dev/disk/by-id/ata-ST4000DM004-XXXX" # devices can be specified using full udev paths
+  #      config: # configuration can be specified at the node level which overrides the cluster level config
+  #    - name: "172.17.4.301"
+  #      deviceFilter: "^sd."
+
+  mon:
```

Comment:
These settings are all in a different order from cluster.yaml. How about if we just copy over the contents of the spec from cluster.yaml for consistent defaults? Otherwise it will be difficult to maintain.